src/main/java/frc/robot/vision/UpdatePose.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.vision;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.drive.DriveSubsystem;
+
+public class UpdatePose extends Command{
+
+  private final VisionSubsystem vision;
+  private final DriveSubsystem drive;
+
+  public UpdatePose(VisionSubsystem vision, DriveSubsystem drive){
+    this.vision = vision;
+    this.drive = drive;
+  }
+
+  @Override
+  public void execute(){
+    if(vision.hasTarget()){
+      drive.resetOdometry(vision.getBotpose());
```

Comment:
I appreciate you making this command. However, we want to use a different method to pass in the botpose. The method is called addVisionMeasurement`.